Src/WitsmlExplorer.Frontend/components/Sidebar/FilterPanel.tsx

```diff
@@ -26,111 +46,128 @@ const FilterPanel = (): React.ReactElement => {
-            autoComplete={"off"}
-          />
-          <FormControlLabel
-            control={
-              <Checkbox
-                id="filter-isActive"
-                onChange={(event) => dispatchNavigation({ type: NavigationType.SetFilter, payload: { filter: { ...filter, isActive: event.target.checked } } })}
-                value={filter.isActive}
-              />
-            }
-            label={"Show only active wells and wellbores"}
-          />
-          <FormControlLabel
-            control={
-              <Checkbox
-                id="filter-objectGrowing"
-                onChange={(event) => dispatchNavigation({ type: NavigationType.SetFilter, payload: { filter: { ...filter, objectGrowing: event.target.checked } } })}
-                value={filter.objectGrowing}
-              />
-            }
-            label={"Show only growing logs"}
-          />
+          <div style={WellboreObject}>
+            <span style={{ display: "flex", gap: "7px" }}>
+              <Typography token={{ fontFamily: "EquinorMedium", fontSize: "0.875rem", color: colors.interactive.primaryResting }}>Limit number of wells</Typography>
+              <Typography token={{ fontStyle: "italic", fontFamily: "EquinorRegular", fontSize: "0.75rem", color: colors.text.staticIconsTertiary }}>(0 for no limit)</Typography>
+            </span>
+
+            <TextField
+              id="filter-wellLimit"
+              type="number"
+              min={0}
+              onChange={(event: ChangeEvent<HTMLInputElement>) =>
+                dispatchNavigation({ type: NavigationType.SetFilter, payload: { filter: { ...filter, wellLimit: Number(event.target.value) } } })
+              }
+              value={filter.wellLimit}
+              autoComplete={"off"}
+            />
+          </div>
           <Divider />
-          <TextField
-            id="curveThreshold-time"
-            label="Set threshold for time curve (in minutes)"
-            type="number"
-            InputProps={{ inputProps: { min: 0 } }}
-            onChange={(event) =>
-              dispatchNavigation({ type: NavigationType.SetCurveThreshold, payload: { curveThreshold: { ...curveThreshold, timeInMinutes: Number(event.target.value) } } })
-            }
-            value={curveThreshold.timeInMinutes}
-            autoComplete={"off"}
-          />
-          <FormControlLabel
-            control={
-              <Checkbox
-                id="curveThreshold-hideInactive"
-                onChange={(event) =>
-                  dispatchNavigation({ type: NavigationType.SetCurveThreshold, payload: { curveThreshold: { ...curveThreshold, hideInactiveCurves: event.target.checked } } })
-                }
-                value={curveThreshold}
-              />
-            }
-            label={"Hide inactive time curves"}
-          />
+          <div style={{ paddingTop: "0.75rem" }}>
+            <Checkbox
+              id="filter-isActive"
```

Comment:
The checkbox becomes unchecked after hiding the FilterPanel, even though isActive is set to true.